### DISCUSSION TITLE
Broken "include src files" link in "Using Panda in a Component Library" guide

### DISCUSSION THREAD

**Discussion by stevezhu** [chakra-ui/panda#2266] [General]
### Subject

Component Library Documentation

### Description

The broken url: https://panda-css.com/docs/guides/component-library#include-the-src-files

"include src files" doesn't link to anything in the [Recommendations ](https://panda-css.com/docs/guides/component-library#recommendations)section:
![image](https://github.com/chakra-ui/panda/assets/4130171/ade24058-865d-4926-8225-caec3ee634e2)

**Comment by astahmer** [comment]
thanks for reporting this ! fixed [here](https://github.com/chakra-ui/panda/pull/2267)
